Dockerfile.alpine

```diff
@@ -25,7 +25,6 @@ RUN make -j2
 RUN mkdir -p ../libtalloc && ar qf ../libtalloc/libtalloc.a bin/default/talloc.c*.o
 ## Compile proot
 WORKDIR /proot
-RUN make -C src loader.elf loader-m32.elf build.h
```

Comment:
I'm not sure if this is needed, but everything appeared to be working (at least in terms of compiling and still producing static binaries) without it.